src/routes/(app)/+page.server.ts

```diff
@@ -40,3 +43,26 @@ export const load = async ({ locals }) => {
 		layoutReset: cmsPage.fullScreen
 	};
 };
+
+export const actions = {
+	navigate: async ({ locals }) => {
+		await collections.sessions.findOneAndUpdate(
```

Comment:
```suggestion
		await collections.sessions.updateOne(
```

you don't use the result